lib/TBDGen/TBDGen.cpp

```diff
@@ -90,6 +90,11 @@ class TBDGenVisitor : public ASTVisitor<TBDGenVisitor> {
       return;
 
     auto declRef = SILDeclRef(VD);
+
+    // Transparent symbols don't exist, even if public.
+    if (declRef.isTransparent())
```

Comment:
Ok, you can just put a FIXME here then. I'll take it out when inlineable things get shared linkage by default.